execute/plugin.go

```diff
@@ -144,6 +156,17 @@ func getPendingExecutedReports(
 			continue
 		}
 
+		// Filter out reports that cannot be executed (snoozed).
+		// TODO: filter func instead of 'canExecute'?
+		var filtered []exectypes.CommitData
+		for _, commitReport := range reports {
+			if !canExecute(commitReport.SourceChain, commitReport.MerkleRoot) {
```

Comment:
Might be worth logging as debug these skipped reports

Reply:
Good idea, added a log.